copyedit elmer:Example uses & Miscellaneous vignette (#192)
tidyverse/elmer: vignettes/elmer.Rmd

## Example uses

> Now that you've got the basic vocab under your belt, I'm going to fire a bunch of interesting potential use cases at you. While many of these have existing special purpose tools that might be faster, cheaper or both, using an LLM allows you to do rapid prototyping, which is extremely valuable even if you do end up using one of those more specialised tools in the final product.

Reviewer: ```suggestion
Now that you've got the basic vocab under your belt, I'm going to fire a bunch of interesting potential use cases at you. While many of these might be solved with special purpose tools that might be faster, cheaper or both, using an LLM allows you to do rapid prototyping, which is extremely valuable even if you do end up using one of those more specialised tools in the final product.
```

?
